middleware/errorMiddleware.js

```diff
@@ -0,0 +1,6 @@
+const errorMiddleware = (err, req, res, next) => {
+    console.error(err.stack); 
+    res.status(500).json({ error: 'Something went wrong!' });
```

Comment:
Add support of the different error codes and custom messages.
Please check the example here: [https://github.com/VitalyaIlin/hillel-code-examples/tree/main/express-middleware](https://github.com/VitalyaIlin/hillel-code-examples/tree/main/express-middleware)